config/sepolia.ts

```diff
@@ -61,7 +61,7 @@ export default <NetworkData>{
             balancerQueriesAddress: '0xe39b5e3b6d74016b2f6a9673d7d7493b6df549d5',
         },
         v3: {
-            vaultAddress: '0xdaa273aeec06e9ccb7428a77e2abb1e4659b16d2',
+            vaultAddress: '0x1fc7f1f84cfe61a04224ac8d3f87f56214fec08c',
```

Comment:
Is that the newest deployment? We'll need to wipe the DB first before adding that...

Reply:
maybe directly in the db? deleting pool records will wipe related records as well